openshift/deploy/statefulset/postgres-workers.yml

```diff
@@ -0,0 +1,133 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+
+parameters:
+  - description: Prefix to prepend to object name.
+    displayName: Prefix
+    name: PREFIX
+    required: true
+  - description: SHA1 of git revision to be deployed.
+    displayName: Git SHA1
+    name: GIT_SHA1
+    required: true
+  - description: Openshift project name
+    displayName: Project
+    name: OC_PROJECT
+    required: true
+  - description: Openshift registry URL
+    displayName: Registry
+    name: OC_REGISTRY
+    required: true
+  - description: Base64-encoded password for the postgres user.
+    displayName: Postgres Password
+    name: POSTGRESQL_PASSWORD
+    required: true
+  - description: Base64-encoded postgres user name.
+    displayName: Postgres User
+    name: POSTGRESQL_USER
+    required: true
+  - description: Base64-encoded postgres database name.
+    displayName: Postgres User
+    name: POSTGRESQL_DBNAME
+    required: true
```

Comment:
`POSTGRESQL_PASSWORD`, `POSTGRESQL_USER` and `POSTGRESQL_DBNAME` are not used and should be deleted.